samples/OldReference/InstrumentedHttpCall.cs

```diff
@@ -20,15 +19,21 @@ public static async Task GetAsync(string url)
             
             Console.WriteLine(string.Join("\n", loaded));
 
-            using (var activity = MyActivitySource.StartActivity("RunAsync"))
+            var activity = new Activity("RunAsync");
```

Comment:
why these changes are done?
the sample was build based on the official sample: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/getting-started/Program.cs